### PR TITLE
Add additional destination height and width parameter to WarpAffine pr…

### PR DESCRIPTION
**Pull request by hojaedev**
…eprocessor

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

File: tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py [hunk 1]
```diff
@@ -480,18 +480,31 @@ def parameters(cls):
...
+                value_type=int,
+                optional=False
             )
         })
         return parameters
 
     def configure(self):
         self.src_landmarks = self.get_value_from_config('src_landmarks')
         self.dst_landmarks = self.get_value_from_config('dst_landmarks')
+        self.dst_height = self.get_value_from_config('dst_height')
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py:505]: @hojaedev do you plan to update existing configs or stay backward compatibility (if parameters are not provided, source height and width will be used)?

Comment (hojaedev) [reply tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py:505]: I will update the configuration for the VAS algorithm that uses warp affine: face recognition.

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py:505]: ok, please do not forget to update

**Pull request merged**